cnxarchive/tests/data/recent.rss

```diff
@@ -269,4 +269,4 @@
       
 
 </channel>
-</rss>
+</rss>
```

Comment:
This is so strange... Why does recent.rss not have the newline? O_o  Is it better to change the test to strip whitespace instead?

Reply:
It might be, but this is working, so I'd rather not fiddle with it.